azkaban-common/src/main/java/azkaban/executor/ExecutionOptions.java

```diff
@@ -33,6 +33,8 @@
   public static final String CONCURRENT_OPTION_SKIP = "skip";
   public static final String CONCURRENT_OPTION_PIPELINE = "pipeline";
   public static final String CONCURRENT_OPTION_IGNORE = "ignore";
+  public static final String FLOW_PRIORITY = "flowPriority";
+  public static final int DEFAULT_FLOW_PRIORITY = 5;
```

Comment:
Do we want to document somewhere saying whether bigger number is higher/lower priority?